### ISSUE TITLE
Ask PCONv a question

### ISSUE THREAD

**Issue by sangsiyuan** [EiriniCharalampakiMsCV2/Wavelet-trasform-for-Image-denoising#1]
Hello, friend:
First of all, thank you for sharing.
I am a student. I pulled your code, but I found that pCONV gave an error when I was running, so I could not run it. Could you please tell me? thank you

**Comment by EiriniCharalampakiMsCV2** [comment]
Yes, you are right! It gives an error, because it misses the function. I have just uploaded all the missing files.
